dev-libs/eb/eb-4.4.3.ebuild

```
@@ -0,0 +1,46 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools
+
+DESCRIPTION="EB is a C library and utilities for accessing CD-ROM books"
+HOMEPAGE="https://web.archive.org/web/20120330123930/http://www.sra.co.jp/people/m-kasahr/eb/"
+SRC_URI="ftp://ftp.sra.co.jp/pub/misc/eb/${P}.tar.bz2"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~alpha amd64 ~arm ~hppa ~ia64 ppc ppc64 ~riscv sparc x86"
+IUSE="ipv6 nls threads"
+
+RDEPEND="
+	sys-libs/zlib
+	nls? ( virtual/libintl )"
+DEPEND="${RDEPEND}"
+BDEPEND="nls? ( sys-devel/gettext )"
+
+DOCS=( AUTHORS ChangeLog{,.0,.1,.2} NEWS README )
+
+PATCHES=(
+	"${FILESDIR}"/eb-4.4-gcc14-iconv.patch #920641
+)
+
+src_prepare() {
+	default
+	eautoreconf
+}
+
+src_configure() {
+	econf \
+		$(use_enable ipv6) \
+		$(use_enable nls) \
+		$(use_enable threads pthread) \
+		--with-pkgdocdir="${EPREFIX}"/usr/share/doc/${PF}/html \
+		--disable-static
```

```suggestion
	local myeconfargs=(
		$(use_enable ipv6)
		$(use_enable nls)
		$(use_enable threads pthread)
		--with-pkgdocdir="${EPREFIX}"/usr/share/doc/${PF}/html
	)

	econf "${myeconfargs[@]}"
```
FYI, `--disable-static` shouldn't be required, it should be passed per default.

---

I'm not sure how to test it.

---

You can simply call `ebuild eb-4.4.3.ebuild configure` and check if `--disable-static` is passed correctly.

For example with 4.4.1 i get (with the same changes)
```
./configure --prefix=/usr --build=x86_64-pc-linux-gnu --host=x86_64-pc-linux-gnu --mandir=/usr/share/man --infodir=/usr/share/info --datadir=/usr/share --sysconfdir=/etc --localstatedir=/var/lib --datarootdir=/usr/share --disable-dependency-tracking --disable-silent-rules --disable-static --docdir=/usr/share/doc/eb-4.4.1-r1 --htmldir=/usr/share/doc/eb-4.4.1-r1/html --with-sysroot=/ --libdir=/usr/lib64 --enable-ipv6 --enable-nls --enable-pthread --with-pkgdocdir=/usr/share/doc/eb-4.4.1-r1/html
```

The `--disable-static` is in the middle. 
Please note that this only works from `EAPI=8`. `EAPI=7` still would need the `--disable-static`.